test/mix/tasks/phoenix.gen.channel_test.exs

```diff
@@ -76,4 +76,10 @@ defmodule Mix.Tasks.Phoenix.Gen.ChannelTest do
       Mix.Tasks.Phoenix.Gen.Channel.run ["Admin.Room", "rooms", "new_message"]
     end
   end
+
+  test "name can't already be defined" do
+    assert_raise Mix.Error, fn ->
+      Mix.Tasks.Phoenix.Gen.Channel.run ["Channel", "channels"]
```

Comment:
If you call it with "Channel", it is going to generate `Phoenix.ChannelChannel` so this test is passing by accident because you are checking the wrong thing. :) Once you do the changes above, this test should fail again.